src/reactjs/logger.native.ts

```diff
@@ -1,7 +0,0 @@
-import type { Logger } from '../core/logger'
```

Comment:
See `getDefaultLogger` in `../core/logger` for what would happen in react-native.

Basically, I went for this version of the logger instead of plain` console` as the default logger just to support both cases. 

If you think it's really necessary to have different loggers for normal/react-native users to save ~50 bytes; sure, I'll do it.